ethereum/contracts/IncentivizedOutboundChannel.sol

```diff
@@ -3,14 +3,33 @@ pragma solidity >=0.7.6;
 pragma experimental ABIEncoderV2;
 
 import "./OutboundChannel.sol";
+import "./DOTApp.sol";
 
 // IncentivizedOutboundChannel is a channel that sends ordered messages with an increasing nonce. It will have incentivization too.
 contract IncentivizedOutboundChannel is OutboundChannel {
 
-    constructor() {
+    uint256 public relayFee;
+    address public feeController;
+    DOTApp private _DOTApp;
```

Comment:
The underscore in `_DOTApp` isn't necessary here, as we've chosen to only use them in function parameter names.

So just `dotApp` instead.